New Article about Getting the word out (#109)
owddm/oktech.jp: content/articles/get-the-word-out/index.md

---
title: Get the word out!
description: Community document on how to get the word out about OKTech!
keywords: ["marketing", "community", "sharing", "contribute", "oktech"]
author: Martin Heidegger <martinheidegger>
date: 2026-06-15
---

Communities are always in flux. People move away, others arrive; older members retire, and younger ones graduate from school.
Because of this, it is important to let the right people know that we are still here and active!

This takes effort, and **we could use your help.**

To ensure we are on the same page, this article explains in detail how you can help spread the word so that new friends can find us and join our community.

### From the heart

> You might not know this, but others really want to know if you enjoy being here! Your experience serves as a signal for newcomers to decide if they could enjoy it too. Being vocal - both in person and on social media  - about the things you enjoy works wonders for the community and would be a huge help to us.

Reviewer: ```suggestion
You might not know this, but others really want to know if you enjoy being here! Your experience serves as a signal for newcomers to decide if they could enjoy it too. Being vocal both in person and on social media about the things you enjoy works wonders for the community and would be a huge help to us.
```
I think this looks cleaner since the dashes look like it's generated from AI and they don't add anything.

* Say thank you to the presenter in Discord.
* Make a social media post about something you enjoyed at an event.
* Write about an idea that inspired you.

Doing these things helps with future planning and gives the people who inspired you a sense of belonging.

### Every event is unique

There are two practical realities we cannot ignore. First, the ways people engage with and work in technology have become much more diverse in recent years; gathering specialists in any one specific niche is even harder now than it was just a few years ago. Second, the number of active English speakers in Kansai is lower than many might expect.

Because of this, when you talk about OKTech, it is important to emphasize that every event will be different and have a different focus.

> The next event might have a completely different vibe or depth. **It helps us immensely if people understand — and embrace — this variety!**

Reviewer: ```suggestion
The next event might have a completely different vibe or depth. **It helps us immensely if people understand and embrace this variety!**
```
I don't think the dashes are adding anything.

In recent years, we have also used LinkedIn and Discord to spread the word.

The best way to grow our Discord is to  ensure it isn't a space that people ignore. Previously, we had a regular update for English-speaking tech events; this was highly valued by members and helped our growth immensely.

* You can volunteer to get those regular updates for English-speaking events in Kansai back into the Discord!
* Help us make the Discord an engaging place rather than a muted one.

LinkedIn is currently our only other active platform. We chose it based on a suggestion and have maintained it since.

* You can help by volunteering to write articles for the LinkedIn page.
* Liking and sharing posts helps us reach new people!

If you are eager to help us grow even further, social media like Facebook is surprisingly still very active in Kansai and would be a great place to mention OKTech.

### Human first, professional second

Using tools like Grammarly or LLMs to ensure your text is free of grammatical errors is a good idea if you feel  insecure about your writing. However, there is charm in imperfection.

> Many people in our group are professionals, but their fields and experiences vary widely. By trying to present ourselves as "strictly professional," we risk creating expectations that we might not meet, which could lead to disappointment.

Reviewer: I would rewrite it in a problem-solution structure:

> By trying to present ourselves as "strictly professional," we risk creating expectations that might not be met, which could lead to disappointment or pressure that would reduce engagement. Therefore making mistakes is fine within our community, since we're all human. The charm to human imperfection makes our interactions feel personal and that is what we like to embrace. 

This will be more appealing and give a better understanding of the point that is being made.

Author: Thanks, added in [849a0f5](https://github.com/oktechjp/oktech.jp/pull/109/commits/849a0f5451d1e77e6bbff47a22d07a7fc6aaba96)

We are not a corporate organization promising career results or academic certifications; we are a community first.

* In your communications, ensure both passion and professionalism are represented.
* While many members are highly skilled, focus on their passion rather than just their credentials.

> * When funds are collected, be transparent about how those proceeds benefit the community as a whole!

Reviewer: ```suggestion
* In your communications, ensure both passion and professionalism are represented.
* Many members are from different backgrounds, focus on their passion and skill rather than just their credentials.
* When contributions are made, be transparent about how those proceeds benefit the community as a whole!
```

- I felt the backgrounds is better aligned with credentials
- Funds sounds very transactional and contributions can also be non-financial
